src/com/rcv/ElectionConfig.java

```diff
@@ -237,6 +237,22 @@ public String electionDate() {
     return rawConfig.date;
   }
 
+  // function: tabulateByPrecinct
+  // purpose: getter for tabulateByPrecinct
+  // returns: true iff we should tabulate by precinct
```

Comment:
"iff" meaning "if and only if" or is this a typo? Probably better to spell it out if it's the former.

Reply:
This seems a bit non-standard?  Not a big deal to me, but it does make me wonder what is being communicated here beyond a regular "if"?